tests/test_adapters.py

```diff
@@ -3,7 +3,7 @@
 
 from pytest import approx, mark, raises
 
-from calculadora_do_cidadao import Igpm, Inpc, Ipca, Ipca15, IpcaE, Selic
+from calculadora_do_cidadao import Igpm, Inpc, Ipca, Ipca15, IpcaE, Selic, Cpiaucsl
```

Comment:
Can we keep the alphabetical order in the `import`s? As the package grows bigger, these minor details can enhance maintainability ; )